proto/provenance/reward/v1/tx.proto

```diff
@@ -0,0 +1,142 @@
+syntax = "proto3";
+package provenance.reward.v1;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/timestamp.proto";
+import "google/api/annotations.proto";
+import "cosmos/base/v1beta1/coin.proto";
+import "provenance/reward/v1/reward.proto";
+
+option go_package          = "github.com/provenance-io/provenance/x/reward/types";
+option java_package        = "io.provenance.reward.v1";
+option java_multiple_files = true;
+
+// Msg
+service Msg {
+
+  // CreateRewardProgram is the RPC endpoint for creating a rewards program
+  rpc CreateRewardProgram(MsgCreateRewardProgramRequest) returns (MsgCreateRewardProgramResponse) {
+    option (google.api.http).post = "/provenance/reward/v1/reward_programs";
+  }
+
+  // EndRewardProgram is the RPC endpoint for ending a rewards program
+  rpc EndRewardProgram(MsgEndRewardProgramRequest) returns (MsgEndRewardProgramResponse) {
+    option (google.api.http).patch = "/provenance/reward/v1/reward_programs/{reward_program_id}";
+  }
+
+  // ClaimRewards is the RPC endpoint for claiming rewards belonging to completed claim periods of a reward program
+  rpc ClaimRewards(MsgClaimRewardsRequest) returns (MsgClaimRewardsResponse) {
+    option (google.api.http).patch =
+        "/provenance/reward/v1/reward_claims/{reward_address}/reward_programs/{reward_program_id}";
+  }
+
+  // ClaimRewards is the RPC endpoint for claiming rewards for completed claim periods of every reward program for the
```

Comment:
The method name in this comment is incorrect

Reply:
good catch, updated comment to reflect ClaimAllRewards